store/conf/attrs/zimbra-attrs.xml

```diff
@@ -10357,4 +10357,9 @@ TODO: delete them permanently from here
   <globalConfigValue>TRUE</globalConfigValue>
   <desc>whether to enable CountObjectsRequest with userAccount/account type on Zimbra Admin Console</desc>
 </attr>
+
+<attr id="4098" name="zimbraFeatureAllowUsernameInPassword" type="boolean" cardinality="single" optionalIn="account,cos,domain" flags="accountInfo,accountCosDomainInherited" since="10.1.0">
```

Comment:
I can see last id for attribute is 4096 so next should be 4097. Any specific reason to avoid this?

Reply:
Ideally if someone is adding a new attribute then next incremented id should be in use irrespective of how many developers are working on same work. Whoever is going to merge code first verify the attribute id and if it conflict update and push the code and then merge.